<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>541</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>761</width>
       <height>531</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="Layout">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 75 20pt &quot;Ubuntu&quot;;
text-align: center; </string>
        </property>
        <property name="text">
         <string>Pan Comido - Inventario</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInventario">
    <property name="title">
     <string>Productos</string>
    </property>
    <addaction name="actionPropio"/>
    <addaction name="actionTerceros"/>
   </widget>
   <widget class="QMenu" name="menuInventario_2">
    <property name="title">
     <string>Inventario</string>
    </property>
    <addaction name="actionDiario"/>
    <addaction name="actionEntradas"/>
   </widget>
   <widget class="QMenu" name="menuReportes">
    <property name="title">
     <string>Reportes</string>
    </property>
    <addaction name="actionGastos"/>
    <addaction name="actionEstadsticas"/>
   </widget>
   <addaction name="menuInventario"/>
   <addaction name="menuInventario_2"/>
   <addaction name="menuReportes"/>
  </widget>
  <action name="action_reporte">
   <property name="text">
    <string>Créditos</string>
   </property>
  </action>
  <action name="actionBuscar">
   <property name="text">
    <string>Buscar</string>
   </property>
  </action>
  <action name="action_cliente">
   <property name="text">
    <string>Mantenimiento</string>
   </property>
  </action>
  <action name="actionBuscar_2">
   <property name="text">
    <string>Buscar</string>
   </property>
  </action>
  <action name="action_producto">
   <property name="text">
    <string>Mantenimiento</string>
   </property>
  </action>
  <action name="actionBuscar_3">
   <property name="text">
    <string>Buscar</string>
   </property>
  </action>
  <action name="action_factura">
   <property name="text">
    <string>Generar</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="action_facturas">
   <property name="text">
    <string>Facturas</string>
   </property>
  </action>
  <action name="action_inventarios">
   <property name="text">
    <string>Inventario</string>
   </property>
  </action>
  <action name="actionTerceros">
   <property name="text">
    <string>Terceros</string>
   </property>
  </action>
  <action name="actionPropio">
   <property name="text">
    <string>Propios</string>
   </property>
  </action>
  <action name="actionD_a_actual">
   <property name="text">
    <string>Día actual</string>
   </property>
  </action>
  <action name="actionSemanal">
   <property name="text">
    <string>Semanal</string>
   </property>
  </action>
  <action name="actionEstad_sticas">
   <property name="text">
    <string>Estadísticas</string>
   </property>
  </action>
  <action name="actionDiario">
   <property name="text">
    <string>Diario</string>
   </property>
  </action>
  <action name="actionEntradas">
   <property name="text">
    <string>Entradas</string>
   </property>
  </action>
  <action name="actionGastos">
   <property name="text">
    <string>Gastos</string>
   </property>
  </action>
  <action name="actionEstadsticas">
   <property name="text">
    <string>Estadísticas</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
